packages/wp-source/package.json

```diff
@@ -21,7 +21,9 @@
     "url": "https://github.com/frontity/frontity/issues"
   },
   "dependencies": {
+    "@frontity/router": "^1.1.1",
     "@frontity/source": "^1.4.0",
+    "@frontity/tiny-router": "^1.2.2",
```

Comment:
This was because I was accessing the `state.router.redirections` in `source.actions.fetch()` here: https://github.com/frontity/frontity/blob/617e400e8c56444a3e773d9db4a9057da73fe87d/packages/wp-source/src/actions.ts#L82

However, I think I actually made a mistake and it is not necessary to include tiny-router as a dependency after all. I could have made and extra null check with the optional chaining operator like: `state?.router?.redirections` and if there is no router, then there are also no redirections. 

But that is all a moot point as we are going to move the redirections to `state.source.redirections` so this dependency can be removed.

Reply:
Yes, I think it can be removed!

But if we were going to keep it I think the dependency should be with `@frontity/router` instead of `@frontity/tiny-router` because if I am not mistaken all you want are the types 🙂

Reply:
> But if we were going to keep it I think the dependency should be with @frontity/router instead of @frontity/tiny-router because if I am not mistaken all you want are the types 🙂

yes, exactly :) I'm not sure why I added the tiny-router as well as the types TBH.

Reply:
Ok. No problem and thanks Michal!